README.rst

```diff
@@ -714,12 +714,12 @@ Code example:
     An ``ImproperlyConfigured`` exception is raised if group already has an
     item registered at ``item_position``.
 
-    You can only register a model link or a generic link.
-    An ``ImproperlyConfigured`` exception is raised
-    if you will try to register a group in place of item.
+    Registeration of a model link or a generic link is only possible.
```

Comment:
Registeration contains a typo, but also doesn't sound great.

What about something like "It is only possible to register links to specific models or generic links" (can you specify what a generic link is? Do you mean an arbitrary/custom URL? If that's the case, I would refer to this as custom URL).

Reply:
I have written generic because we have a subheading ``Adding a generic link``. It will be easier to relate.

Reply:
> I have written generic because we have a subheading `Adding a generic link`. It will be easier to relate.

Ok, I would still spend a few more words to explain generic link means any URL the user wants, with any word, no strong opinion on this.

Reply:
I think then custom sounds better